dapps/evancore.vue.libs/src/components/dapp-wrapper/dapp-wrapper.ts

```diff
@@ -46,6 +47,8 @@ const i18nPref = '_evan._routes';
  * navigation, that can be applied by every component. Have a look at the breadcrumbs /
  * dapp-wrapper-level-2 component.
  *
+ * TODO: Rework login function
```

Comment:
What needs to be reworked?

Reply:
It's really confusing what the login function does or is. It shouldn't become `null` somehow. We can discuss this in detail later on.